todo_app/data/api_client.py

```diff
@@ -0,0 +1,13 @@
+import os
+import requests
+
+def get_cards_request():
+    result = requests.get(f'https://api.trello.com/1/boards/{os.getenv("BOARD_ID")}/lists?cards=open&key={os.getenv("SECRET_KEY")}&token={os.getenv("TOKEN")}')
+    if (result.status_code == 200):
+        return result.json()
```

Comment:
Not necessarily diagreeing, but is it right that this just returns nothing if the result has failed? Alternative patterns for the same might be to check if it's not responded correctly, and if not then error or handle it. There are also some nice patterns built into the responses that `requests` returns, such as:
```python
    if result.ok: # will still work if they return a 2XX
        return result.json()
```

```python
    # will raise an exception if the request errored
    result.raise_for_status() 
    return result.json()
```